frontend/css/layout.css

```diff
@@ -64,25 +79,119 @@ li a {
     padding-top: var(--spacing-m);
 }
 
-.ideas {
-    text-align: center;
+.nav__icons__item {
```

Comment:
I'd probably move these up to be defined together with the block element `.nav` above (and possibly move them into a separate file). 
Again, if you'd prefer to do further refinements in a separate issue, that's totally fine with me.